funnel/models/project.py

```diff
@@ -88,7 +88,9 @@ class Project(BaseScopedNameMixin, db.Model):
     labels = db.Column(JsonDict, nullable=False, server_default='{}')
 
     featured_sessions = db.relationship("Session",
-        primaryjoin="and_(Session.project_id == Project.id, Session.featured == True)", back_populates="project")
+        primaryjoin="and_(Session.project_id == Project.id, Session.start != None, Session.end != None, Session.featured == True)", back_populates="project")
```

Comment:
Unscheduled sessions can be featured. That was the main motivation for having unscheduled sessions.